src/components/CohortBuilder/Summary/Cards/SummaryCard.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement, useRef, RefObject } from 'react';
+import { Card as AntdCard } from 'antd';
+import './SummaryCard.css';
+
+type CardProps = {
+  title?: string;
+  loading?: boolean;
+  children: ReactElement[] | any | Function;
+};
+
+const Card = ({ title, loading, children, ...props }: CardProps) => {
```

Comment:
can we make ...props explicit or at least as explicit as possible? By explicit I mean to replace ...props with all the values it represents

Reply:
in that case, the component should not know about it, since it's passing it around for the next component.

I know we have issue in the code about over using it. But I think It's a case where it make sense to do it.